src/App.js

```diff
@@ -4,9 +4,30 @@ import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export class App extends Component {
-  state = {};
+  state = {
+    query: '',
+  };
+
+  moviesFilter = (query) => {
```

Comment:
есть разница в перестановки слов
в моем случае это действие

фильмыФильтер
отфильтроватьФильмы
```suggestion
  filterMovies = (query) => {
```